MeshPipe/MeshPipe.m

```diff
@@ -7,25 +7,30 @@
 //
 
 #import "MeshPipe.h"
-#import "AsyncUdpSocket.h"
+#import "GCDAsyncUdpSocket.h"
 #import "GZIP.h"
+#include <netinet/in.h>
+#include <arpa/inet.h>
+static NSString *MeshPipeLocalAddress = @"127.0.0.1";
 
-//#define MPDebug(...) NSLog(__VA_ARGS__)
 #define MPDebug(...)
```

Comment:
This should yield a "redefinition" warning in Release builds; put it in an #elif plz